client/verta/tests/test_endpoint/test_resources.py

```diff
@@ -4,14 +4,15 @@
 from verta.endpoint.update import DirectUpdateStrategy
 
 
-@pytest.mark.parametrize("data", [3, 64, 0.25])
+@pytest.mark.parametrize("data", [(3, DirectUpdateStrategy()), (64, None),
+                                  (0.25, DirectUpdateStrategy())])
 def test_cpu_milli(client, data, in_tempdir):
```

Comment:
This should work, and be clearer to read.
```suggestion
@pytest.mark.parametrize("data,strategy", [(3, DirectUpdateStrategy()), (64, None),
                                  (0.25, DirectUpdateStrategy())])
def test_cpu_milli(client, data, strategy, in_tempdir):
```

Reply:
thank you, yes. I've updated it